org.mwc.debrief.lite/src/test/java/org/mwc/debrief/lite/tests/TestLiteLaunch.java

```diff
@@ -0,0 +1,76 @@
+/*
+ *    Debrief - the Open Source Maritime Analysis Application
+ *    http://debrief.info
+ *
+ *    (C) 2000-2016, Deep Blue C Technology Ltd
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the Eclipse Public License v1.0
+ *    (http://www.eclipse.org/legal/epl-v10.html)
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+ */
+package org.mwc.debrief.lite.tests;
+
+import java.awt.Component;
+
+import javax.swing.JFrame;
+
+import org.mwc.debrief.lite.DebriefLiteApp;
+import org.mwc.debrief.lite.utils.TestUtils;
+import org.pushingpixels.flamingo.api.ribbon.JRibbonFrame;
+
+/**
+ * @author Ayesha
+ *
+ */
+public class TestLiteLaunch extends BaseTestCase
+{
+
+  public void testTitleAndIcon()
+  {
+    JFrame ribbonFrame = DebriefLiteApp.getInstance().getApplicationFrame();
+    assertNotNull(ribbonFrame.getIconImage());
+    assertNotNull(ribbonFrame.getTitle());
+    assertTrue(ribbonFrame.getTitle().contains("Debrief Lite"));

```

Comment:
I _think_ it would be better to refer to the original field (if present) that contains "Debrief Lite", rather than to use this _magic string_.